README.md

```diff
@@ -23,29 +23,33 @@ Parsers are ordered and takes specific input to generate specific output. This w
 
 By using parsers, you dictate the way you receive the data from Specify to fit your own needs.
 
-## Creating your own parser
+## How to create your own parser?
 
-1. Git clone this repo
-2. In the directory `parsers`, create a directory with your parser's name
-3. Create your valid parser
-4. Make a PR
+Let's say you want to create a parser named `my-parser`.
+
+1. Fork the current repository
+2. Git clone this repo
+3. In the directory `parsers`, create a directory named `my-parser`
+4. Create your valid parser
+5. Make a PR
 
 ### Creating a valid parser
 
-To be valid, a parser needs:
-- A `README.md` file which contains:
+To be valid, your parser needs:
+- A `README.md` file which containing:
```

Comment:
`containing` or `which contains`, not both

Reply:
DOne